internal/providers/terraform/azure/testdata/security_center_subscription_pricing_test/security_center_subscription_pricing_test.golden

```diff
@@ -6,10 +6,10 @@
  └─ Defender for servers, plan 2                                                                            Monthly cost depends on usage: $14.60 per server           
                                                                                                                                                                        
  azurerm_security_center_subscription_pricing.standard_example["AppServices"]                                                                                          
- └─ Defender for app service                                                                                Monthly cost depends on usage: $14.60 per node             
+ └─ Defender for app service                                                                                Monthly cost depends on usage: $14.60 per nodes            
```

Comment:
Can we keep the units singular? Otherwise `per Xs` (for other units too) doesn't read right.

Reply:
Can change it back, but it's inconsistent with what we do with other units.